### ISSUE TITLE
User Feedbacks

### ISSUE THREAD

**Issue by adamfaliq42** [jivesearch/jivesearch#27]
I showed Jive Search to my friends and family (roughly ~20 people) and here are the feedbacks 👍 

👎 

1. Unsatisfying search result
- search tends to produce reference to a website rather than its actual address

2. Upvote button not clear
- Some users didn't see the upvote button

3. "About Us" page is not optimised for mobile browser 

![screenshot_20180503_111403](https://user-images.githubusercontent.com/27236918/39571445-6a68014e-4ec3-11e8-810d-7ffdf1eddfa3.jpg)

👍 

1. Simple design
2. Results are fast

**Comment by brentadamson** [comment]
1. We need to basically rip out the crawler I wrote and switch to Colly. Colly came out after I wrote my crawler but b/f I put the project up on Github. https://github.com/gocolly/colly. 
2. Once we get people to vote I think it will become clearer (I think?) as the "0" will turn into different numbers and catch your eye more.
3. Nothing is really optimized for mobile yet. The search bar doesn't look good, width is off on search results, Wikipedia summary looks bad. Lots to do there.

Can we move this one to chat? https://discord.gg/cfxQkuh
